Dockerfile

```diff
@@ -1,9 +1,5 @@
-FROM google/dart:2.5 as dart2
-FROM google/dart:1.24.3
-
+FROM google/dart:2.5
 WORKDIR /build/
-ADD pubspec.yaml /build
-COPY --from=dart2 /usr/lib/dart /usr/lib/dart2
-RUN _PUB_TEST_SDK_VERSION=1.24.3 /usr/lib/dart2/bin/pub get --no-precompile
-ARG BUILD_ARTIFACTS_BUILD=/build/pubspec.lock
```

Comment:
Shouldn't we keep `pubspec.lock` as an artifact? If no what is the purpose of `pub get`?

Reply:
Workiva-build now auto-discovers all `pubspec.lock` artifacts so this is no longer necessary. That's something we can add to the cleanup doc – if we're specifying an artifact for pubspec.locks, we can remove those

Reply:
The Workiva Build system scans for `pubspec.lock` files, so these explicit artifacts exposing the lock files are unnecessary.

Reply:
Cool